02-microarray/pathway-analysis_microarray_01_ora.Rmd

```diff
@@ -326,15 +345,15 @@ Over-representation testing using `clusterProfiler` is based on a hypergeometric
 
 Where `N` is the number of genes in the background distribution, `M` is the number of genes in a pathway, `n` is the number of genes we are interested in (our differentially expressed genes), and `k` is the number of genes that overlap between the pathway and our genes of interest.
 
-So, we will need to provide to `clusterProfiler` two genes lists: 
+So, we will need to provide to `clusterProfiler` two genes lists:
 
-1) Our genes of interest (`n`) 
-2) What genes were in our total background set (`N`). (All genes that originally had an opportunity to be measured). 
+1) Our genes of interest (`n`)
+2) What genes were in our total background set (`N`). (All genes that originally had an opportunity to be measured).
 
 ## Determine our genes of interest list
 
-We will use our differential expression results to get a genes of interest list. 
-Let's use our adjusted p values as a cutoff. 
+We will use our differential expression results to get a genes of interest list.
+Let's use our adjusted p values as a cutoff.
```

Comment:
May be good to mention in the paragraph here that the code also filters out down regulated genes in case the reader just skims over the code and misses that detail

Reply:
Gonna also put this on the docket for a PR with ORA specific changes along with #306